nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/test/java/org/nuxeo/ecm/automation/core/operations/document/AddEntryToMultiValuedPropertyTest.java

```diff
@@ -118,10 +167,11 @@ public void shouldFailedIfToTypeNotMatch() throws Exception {
         operation.value = Boolean.FALSE;
 
         try {
+            operation.xpath = "lists:integer";
             operation.run(doc);
             fail();
-        } catch (UnsupportedOperationException e) {
-            assertEquals("Given type \"false\" value is not a string type", e.getMessage());
+        } catch (OperationException | UnsupportedOperationException e) {
+            assertEquals("Given type \"false\" value is not a long type", e.getMessage());
         } catch (Exception e) {
             fail();
```

Comment:
It know it's part of the original code but it's better to clean it up. Remove these two lines, just let the Exception be thrown, it'll give clearer test error messages if there's a failure.

Reply:
The goal was to test specific exception: as you can see in the catch Exception, there is a fail. I can remove it.

Reply:
Yes the idea is that if you remove `catch (Exception e) { fail(); }` you'll get clearer exceptions in case of failure.